pkg/client/proxy.go

```diff
@@ -40,6 +39,7 @@ type ServiceContext struct {
 	quit context.CancelFunc
 
 	service rpc.ProxyEngineServiceClient
+	health  healthpb.HealthClient
```

Comment:
Off the topic:
Just notice the clients of process manager and instance don't have `heath`.
Is it better to add it into the two clients as well?
cc @ejweber @innobead

Reply:
If there is a healthy server implemented. this sounds good to me.